packages/integration-tests/api/distribution.js

```diff
@@ -10,16 +10,32 @@ const got = require('got');
  * @param {string} accessToken
  *   Access token from OAuth provider
  *
- * @returns {string}
+ * @returns {Response} - response object from distribution api
  *   S3 signed URL to access file protected by distribution API
  */
-async function getDistributionApiS3SignedUrl(fileUrl, accessToken) {
+async function getDistributionApiResponse(fileUrl, accessToken) {
   const response = await got(fileUrl, {
```

Comment:
I know this is my code but I wonder why I was doing it. You could just `return got(....` and it would have the same effect as await and return

Reply:
I think this is actually my code, and you're probably right.

Reply:
7648a83fb